src/services/services.ts

```diff
@@ -3705,6 +3766,39 @@ namespace ts {
                 return entries;
             }
 
+            function getJsDocCompletionEntries(fileName: string, position: number): CompletionEntry[] {
+                let sourceFile = getValidSourceFile(fileName);
+                let hasJsDocComment = ts.hasDocComment(sourceFile, position);
+                if (!hasJsDocComment) {
+                    return undefined;
+                }
+
+                // Return a list of JsDoc tag names for completion if the current position is right after an '@' sign 
+                if (sourceFile.text.charCodeAt(position - 1) === CharacterCodes.at) {

```

Comment:
it shouldn't be "right after".  for example if I have  "@t" and I try to bring up completion, we'll want that to work.  Instead, you want to know if the previous token (prior to the one you might be writing) is an @ sign.